### PR TITLE
[51] Develop a Contact Form Page using HTML

### PR DESCRIPTION
**Pull request by DamisiM**
## Task: `[51] Develop a Contact Form Page using HTML`
- `Created a contact page
IMG_20201011_155921
IMG_20201011_155935

`

### CODE REVIEW

File: submissions/damisiMahmood/html-forms/Contact Page.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<!Doctype html>
```
Comment (adaugochi) [inline submissions/damisiMahmood/html-forms/Contact Page.html:1]: Remove spaces in this file name

**Review by adaugochi — CHANGES_REQUESTED** [review]
Images are not needed here. Kindly remove them

Also, remove the Benue1-20-Hack1 that is part of this PR

File: submissions/damisiMahmood/html-forms/Contact Page.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            <br>
+            <label for="subject">Subject</label> <br>
+            <input type="text" name="subject" id="Subject" size="40" maxlength="30"/><br> <br>
+            <label for="message">Message</label> <br>
+            <textarea rows="7" cols="50" name="Message" maxlength="300"></textarea>
+            <br>
+            <br>
+
+            <button type="submit">Submit</button>
+        </fieldset>
```
Comment (TundeOladejo) [inline submissions/damisiMahmood/html-forms/Contact Page.html:38]: The closing tag for the fieldset should not be inside the form element tag. Move it.

File: submissions/damisiMahmood/html-forms/Contact Page.html [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        <br>
+    <footer> 
+        <section>
+            <img src="brandslogo.png" width="1200" height="400" alt="OUR BRANDS">
+        </section>
+        <p><strong>Contact us @ I-Damsels Avenue, London</strong>
+            <ul> Call us:
+                <li>Nigeria: +234 xxxxxxxxxx</li>
+                <li>UK:      +1 xxxxxxxxxx</li>
+                <li>US:      +2 xxxxxxxxxx</li>
```
Comment (TundeOladejo) [inline submissions/damisiMahmood/html-forms/Contact Page.html:49]: Remove extra spaces

**Review by TundeOladejo — CHANGES_REQUESTED** [review]

File: submissions/damisiMahmood/html-forms/Contact Page.html (hunk 3 above)
Comment (DamisiM) [reply submissions/damisiMahmood/html-forms/Contact Page.html]: Done

File: submissions/damisiMahmood/html-forms/Contact Page.html (hunk 2 above)
Comment (DamisiM) [reply submissions/damisiMahmood/html-forms/Contact Page.html]: Done

File: submissions/damisiMahmood/html-forms/Contact Page.html (hunk 1 above)
Comment (DamisiM) [reply submissions/damisiMahmood/html-forms/Contact Page.html]: Done

Comment (DamisiM) [reply submissions/damisiMahmood/html-forms/Contact Page.html]: Adaa, the images are a part of my website, they are in the header and footer of my website.

Kindly note

**Review by adaugochi — CHANGES_REQUESTED** [review]
Place all images inside the `images` folder that is in your `html` folder

**Review by adaugochi — CHANGES_REQUESTED** [review]
Kindly place your images inside the image folder that is in your HTML forlder

**Comment by DamisiM** [comment]
I want my images to be in the same folder as the task, you will have to explain the essence of placing it in a separate folder

**Comment by ToluAkin** [comment]
You want to have all your images in your HTML folder because all your images should be in one place. Since you first created the images folder in HTML folder, then all the images add thereafter should be in html/images folder. Do you get @DamisiM ?

**Comment by DamisiM** [comment]
@adaugochi Kindly review, I have made the required changes

File: submissions/damisiMahmood/html-forms/contactPage.html [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+            <br>
+            <br>
+
+            <button type="submit">Submit</button>
+        </form>
+        </fieldset>
+        <br>
+    <footer> 
+        <section>
+            <img src="C:\Users\peter\Desktop\Dufuna-CodeCamp20\submissions\damisiMahmood\html\images/brandslogo.png" width="1200" height="400" alt="OUR BRANDS">
```
Comment (adaugochi) [inline submissions/damisiMahmood/html-forms/contactPage.html:43]: This is not this best way to target a path. Just imagine I clone your work. Of course, my system won't have this path `C:\Users\peter\`. Kindly, used `OUR BRANDS`

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/damisiMahmood/html-forms/contactPage.html (hunk 4 above)
Comment (DamisiM) [reply submissions/damisiMahmood/html-forms/contactPage.html]: This has been reviewed. Kindly review. Thank you

**Review by adaugochi — APPROVED** [review]

**Pull request merged**